src/lib/operations/create-user/createUser.ts

```diff
@@ -24,17 +25,47 @@ export async function createUser(
   // TODO: Notify only once the keys are generated and/or the Auth0 user is updated?
   notifyUserCreationOnSlack(idToken);
 
-  // Wait for the key to be generated...
-  // TODO: Replace with code that actually checks the key state:
-  await delay(2000);
+  /*
```

Comment:
This is now commented out as I first want to see the errors logged from `createUser.utils.ts` to do a couple small changes, and then I'll uncomment this.